### ISSUE TITLE
Unable to navigate to preview when editing an item

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#4026]
Using Voiceover, I am unable to navigate to the eye icon when editing an item.

<img width="897" height="54" alt="Image" src="https://github.com/user-attachments/assets/61827966-38a5-4d38-8fe9-b0f6ae109f46" />
